test/helpers/react_components/mentoring/session/exemplar_files_test.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "../../../../../app/helpers/react_components/mentoring/session/exemplar_file_list"
```

Comment:
I'm testing without requiring Rails. This will make our test suite faster.

Reply:
I'm not sure you actually are. If you don't have Rails then `ActiveSupport::TestCase` below won't exist so it'll fail. I think you're just not referencing test_helper.rb here, which means if you run this file by itself, it'll break. When you run the whole suite, the test_helper will be required for any file, so the whole thing will render.

Reply:
So changing this back to require `test_helper` and then running results in this:

<img width="614" alt="Screenshot 2021-09-22 at 16 30 47" src="https://user-images.githubusercontent.com/286476/134374143-2e89b078-2950-4a91-bc75-6ebb81ee53f1.png">

So it's already very fast when you just run this as one file. All the slowdown really is when we start to do stuff with databases etc.

Reply:
@kntsoriano I've reverted this for now. I'm not against doing things like this, but its important we do them systematically across everything rather than just as a one-off variation. So let's work out what we can do to speed everything up and then implement everywhere :)

Reply:
It runs a lot faster without requiring the test helper:
![Screen Shot 2021-09-27 at 7 10 05 AM](https://user-images.githubusercontent.com/1901520/134827305-1564228d-13e5-4264-9a68-20ba156df0fb.png)

I know that we want to be systematic about things, but having really slow tests and fixing tests that break when we don't want them really makes it frustrating from day to day.

Reply:
> I know that we want to be systematic about things, but having really slow tests and fixing tests that break when we don't want them really makes it frustrating from day to day.

Totally agree with this. But `0.005s` is pretty fast already. Our whole test suite would run in 2s if all the tests were that fast. I don't think this is where the speed improvements are going to come from. There's many tests that take 5s to run, which are 1000x slower than this, which is where I think we need to find solutions.

Reply:
The difference is though is when you TDD and run the tests constantly. This is where the difference lies in getting instant feedback versus waiting another multiple of a second. I find that I get to focus easier and I get work done a lot faster.

Reply:
Yeah, that makes sense.